Nextras/YoutubeApi/Reader.php

```diff
@@ -68,14 +68,14 @@ public function getVideo($videoId)
 	protected function getData($videoId)
 	{
 		$url = "https://www.googleapis.com/youtube/v3/videos?key={$this->apiKey}&part=snippet,contentDetails&id={$videoId}";
-		$curl = new CurlWrapper($url, Nette\Http\Request::GET);
-		$curl->setOption('CAINFO', CertificateHelper::getCaInfoFile());
+		$client = new GuzzleHttp\Client;
+		$request = $client->request('GET', $url, ['verify' => CertificateHelper::getCaInfoFile()]);
 
-		if (!($response = $curl->execute())) {
-			throw new \RuntimeException('Unable to parse YouTube video: ' .  $curl->error);
+		if ($request->getStatusCode() != '200') {
```

Comment:
This will never trigger as guzzlehttp throws exception by default and requires `http_errors` set to false http://docs.guzzlephp.org/en/latest/request-options.html#http-errors

Also, I believe psr7 guarantees int status https://github.com/guzzle/psr7/blob/master/src/Response.php#L79